src/test/java/edu/umich/lib/dor/replicaexperiment/ReplicaExperimentApplicationTests.java

```diff
@@ -92,25 +92,20 @@ class ReplicaExperimentApplicationTests {
     RepositoryClient repoOneClient;
     RepositoryClient repoTwoClient;
 
-    void resetDirPath(Path path) {
+    void resetDirPath(Path path) throws IOException {
         if (Files.exists(path)) {
-            try {
-                FileSystemUtils.deleteRecursively(path);
-            } catch (IOException e) {
-                log.error(e.getMessage());
-            }
-        }
-        try {
-            Files.createDirectories(path);
-        } catch (IOException e) {
-            log.error(e.getMessage());
+            FileSystemUtils.deleteRecursively(path);
         }
+        Files.createDirectories(path);
     }
 
     @BeforeEach
-    void init() {
+    void init() throws IOException {
         resetDirPath(repoOneStoragePath);
+        resetDirPath(repoOneWorkspacePath);
         resetDirPath(repoTwoStoragePath);
+        resetDirPath(repoTwoWorkspacePath);
```

Comment:
This makes sure tests will run without issue. There might have been an error before because `workspace` directories didn't exist.